src/test/java/org/jenkinsci/plugins/workflow/cps/CpsThreadTest.java

```diff
@@ -67,7 +67,8 @@
         List<CauseOfInterruption> causes = iba.getCauses();
         assertEquals(1, causes.size());
         assertEquals(CauseOfInterruption.UserInterruption.class, causes.get(0).getClass());
-        r.assertLogContains("never going to stop", b); // TODO under heavy load sometimes fails here; why?
+        r.waitForMessage("Finished: ABORTED", b); // TODO WorkflowRun.isBuilding() can go to false before .finish has completed
```

Comment:
I do not know of any impact on users—we only see it because `waitForCompletion` goes into a fast spin loop and so might proceed with the test during the instant before the log is closed. But I will file a bug for it just in case.